docs/DeveloperGuide.adoc

```diff
@@ -96,22 +96,22 @@ When you are ready to start coding,
 [[Design-Architecture]]
 === Architecture
 
-.Architecture Diagram
-image::Architecture.png[width="600"]
+The *_Architecture Diagram_* (Refer to Figure 1) explains the high-level design of the App. Given below is a quick overview of each component.
```

Comment:
Can we change the format to something like "The Architecture Diagram shown in _Figure 1_ explains....."?
Also, should we add links (in prose) to all figures or just the ones that are not in close proximity?

Reply:
Some texts refer to the figures like this -```(Refer to Figure 1)```, while some of them have a combination of this and ```The XXXX Diagram shown in Figure 1```. I don't feel this is inconsistent simply because this combination is used widely.